leapp/utils/report.py

```diff
@@ -18,3 +18,19 @@ def fetch_upgrade_report_messages(context_id):
         messages.append(report)
 
     return messages
+
+
+def generate_report_file(messages_to_report, path):
+    if path.endswith(".txt"):
+        with open(path, 'w') as f:
+            for message in messages_to_report:
+                f.write('Severity: {}\n'.format(message['severity']))
+                f.write('Title: {}\n'.format(message['title']))
+                f.write('Summary: {}\n'.format(message['summary']))
+                if message.get('detail', {}).get('remediations'):
+                    f.write('Remediation: {}\n'.format(','.join(["{}: {}".format(r['type'], r['context'])
+                                                                 for r in message['detail']['remediations']])))
+                f.write('-' * 40 + '\n')
+    elif path.endswith(".json"):
+        with open(path, 'w') as f:
+            json.dump({'entries': messages_to_report}, f, indent=2)
```

Comment:
Can we also get here the UUID of the run as stored in DB?

```suggestion
            json.dump({'entries': messages_to_report, 'leapp_run_id': id}, f, indent=2)
```